scripts/corelli/calibration/bank.py

```diff
@@ -180,10 +186,83 @@ def collect_bank_fit_results(output_workspace: str,
     return workspace
 
 
-def criterium_peak_pixel_position(peak_table: InputTable,
+def criterion_peak_vertical_position(peak_table: InputTable,
+                                     summary: Optional[str] = None,
+                                     zscore_threshold: float = 2.5,
+                                     deviation_threshold: float = 0.0035) -> np.ndarray:
```

Comment:
I'd just add a note on hardcoded defaults

Reply:
I described in the docstring that `0.0035` corresponds to the height of three CORELLI pixels